lib/constants.js

```diff
@@ -284,102 +284,6 @@ CapacityMode.ON_DEMAND = new CapacityMode(2);
 
 CapacityMode.seal();
 
-
```

Comment:
Does the move of sync policies from here to durability.js change the public API?

Reply:
It shouldn't, since the user gets everything from index.js.  I changed it to get durability-related classes from durability.js.